r_basics_transform_data/r_basics_transform_data.md

```diff
@@ -253,13 +262,21 @@ The difference between `=` and `==`
 
 One common issue to be aware of is the difference between the single equals (=) and the double equals (==) operators.
 
-In R, using a single equals sign assigns a value.  It asserts "these things **are** equal."
+In R, using a single equals sign **assigns a value**.  It asserts "these things **are** equal."  
+
+* For example, `data = my_data` means "please set `data` equal to `my_data`."  
+
+The double equals sign does not assign, but **compares**.  It asks "**are** these things equal?".  That's why we use double equals in the context of a logical test that **compares** the left hand side, e.g. `mrn`, with the right hand side, e.g. 5000083, to check whether or not they are the same.
```

Comment:
```suggestion
The double equals sign does not assign, but **compares**.  It **asks** "are these things equal?".  That's why we use double equals in the context of a logical test that **compares** the left hand side, e.g. `mrn`, with the right hand side, e.g. 5000083, to check whether or not they are the same.
```

Reply:
This is a little change, and I'm not a attached to it-- I just think that changing the words that are bolded here drives home the difference a little better. Feel free to leave it the way you had it if you disagree!